cmd/root.go

```diff
@@ -52,20 +53,31 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.kfn.yaml or $(pwd)/.kfn.yaml")
-	rootCmd.PersistentFlags().BoolVarP(&Verbose, "verbose", "v", false, "verbose output")
-	rootCmd.PersistentFlags().String("registry", "", "Docker registry where to push the image")
-	viper.BindPFlag("registry", rootCmd.PersistentFlags().Lookup("registry"))
-	rootCmd.PersistentFlags().String("registry-username", "", "Username to access docker registry")
-	viper.BindPFlag("registry_username", rootCmd.PersistentFlags().Lookup("registry-username"))
-	rootCmd.PersistentFlags().String("registry-password", "", "Password to access docker registry")
-	viper.BindPFlag("registry_password", rootCmd.PersistentFlags().Lookup("registry-password"))
 
-	rootCmd.PersistentFlags().String("kubeconfig", "", "kubeconfig")
-	viper.BindPFlag("kubeconfig", rootCmd.PersistentFlags().Lookup("kubeconfig"))
+	rootCmd.PersistentFlags().BoolP(config.VERBOSE, "v", false, "verbose output")
+	viper.BindPFlag(config.VERBOSE, rootCmd.PersistentFlags().Lookup(config.VERBOSE))
+
+	stringFlagWithBind(config.REGISTRY, "", "Docker registry where to push the image")
+	stringFlagWithBind(config.REGISTRY_USERNAME, "", "Username to access docker registry")
+	stringFlagWithBind(config.REGISTRY_PASSWORD, "", "Password to access docker registry")
```

Comment:
I was going to ask why you needed these, but after our Slack conversation today it's become clear! :)